### PR TITLE
Grades views

### PR DESCRIPTION
**Pull request by mwitte98**
Related Issue #28 

Changes:
- Grades views displaying your own grades and course grades for instructors

### CODE REVIEW

File: client/src/components/AppBarRight.tsx [hunk 1]
```diff
@@ -34,20 +35,27 @@ export class AppBarRight extends React.Component<AppBarRightProps, PopoverState>
...
 
     handleLogoutClose = () => {
         this.setState({
             open: false,
         });
         this.props.onLogoutClick ? this.props.onLogoutClick() : '';
-    };
+    }
+
+    handleGradesClose = () => {
```
Comment (djrenren) [inline client/src/components/AppBarRight.tsx]: This is the same function as handleLogoutClose and presumably always would be. Any chance we can redo the semantic and just handleMenuClose?

**Review by djrenren — CHANGES_REQUESTED** [review]
Just the one feedback. otherwise looks good.

**Comment by mwitte98** [comment]
@djrenren I wanted to combine those methods initially but couldn't figure out how to. Finally found a way that works. I improved backend performance as well.

**Review by djrenren — APPROVED** [review]
B-E-A-UTIFUL

**Pull request merged**